game/src/ui/ability_widget.rs

```diff
@@ -61,12 +61,8 @@ impl<'w, 's> UiAbilityWidgetExt<'w, 's> for UiBuilder<'w, 's, '_, Entity> {
                         let handle =
                             ui_materials.add(ability_cooldown::Material {
                                 factor: 0.3,
-                                background_color: Color::rgba(
-                                    0.0, 0.0, 0.0, 0.0,
-                                )
-                                .into(),
-                                filled_color: Color::rgba(0.0, 0.0, 0.0, 0.6)
-                                    .into(),
```

Comment:
Removed the conversions here since both `background_color` and `filled_color` are of `Color` type.